grafana/resource_organization_test.go

```diff
@@ -126,6 +126,18 @@ func TestAccOrganization_users(t *testing.T) {
 					),
 				),
 			},
+			{
+				Config: testAccOrganizationConfig_ignoreMissingUsers,
+				Check: resource.ComposeTestCheckFunc(
+					testAccOrganizationCheckExists("grafana_organization.test", &org),
+					resource.TestCheckResourceAttr(
+						"grafana_organization.test", "name", "terraform-acc-test-missing-user",
+					),
+					resource.TestCheckResourceAttr(
+						"grafana_organization.test", "admins.#", "1",
```

Comment:
point to discuss: currently the state contains exactly what have been provided by the user but as we prune the list when `ignore_missing_users=true` should we adjust what is stored in the state?
note: The simpler is to not modify the content of the state as it adds more code logic and might not be counter intuitive.

The test is left failing purposely to avoid merging this without having the discussion.

The same logic should be applied for resource_team as well.